src/storm/utility/ConstantsComparator.cpp

```diff
@@ -159,11 +159,11 @@ namespace storm {
         template class ConstantsComparator<storm::storage::sparse::state_type>;
 
 #ifdef STORM_HAVE_CARL
-#if defined(STORM_HAVE_CLN)
+#if defined(STORM_HAVE_CLN) && !defined(STORM_USE_CLN_EA)
```

Comment:
I'm slightly confused. Why do we instantiate the comparator for CLN only if we are _not_ using CLN (for exact arithmethic)?

Reply:
I don't know, but it is used in other compilation units i.e. gives linkage errors if not instantiated. As I said I suspect a bug there, as this instantiation would use the template and not an explicit specialisation.

Reply:
This commit should not change the current behavior but I do think the current behavior is buggy...